src/ProfileListBuilder.php

```diff
@@ -51,13 +59,25 @@ class ProfileListBuilder extends EntityListBuilder {
    *   The date formatter service.
    * @param \Drupal\Core\Render\RendererInterface $renderer
    *   The renderer.
+   * @param \Drupal\Core\Routing\RedirectDestinationInterface $redirect_destination
+   *   The redirect destination service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   The entity type manager.
    */
-  public function __construct(EntityTypeInterface $entity_type, EntityStorageInterface $storage, DateFormatter $date_formatter, RendererInterface $renderer, RedirectDestinationInterface $redirect_destination) {
+  public function __construct(
+    EntityTypeInterface $entity_type,
+    EntityStorageInterface $storage,
+    DateFormatter $date_formatter,
+    RendererInterface $renderer,
+    RedirectDestinationInterface $redirect_destination,
+    EntityTypeManagerInterface $entityTypeManager
```

Comment:
Variables are named in `snake_case` instead of `camelCase`, unlike object properties.
Should read `$entity_type_manager` like the rest.